Mathlib/MeasureTheory/MeasurableSpace/Basic.lean

```diff
@@ -755,11 +755,21 @@ theorem measurable_from_prod_countable' [Countable β]
   rw [this]
   exact .iUnion (fun y ↦ (hf y hs).prod (.measurableAtom_of_countable y))
 
+/-- For the version where the first space in the product is countable,
+see `measurable_from_prod_countable''`.-/
 theorem measurable_from_prod_countable [Countable β] [MeasurableSingletonClass β]
     {_ : MeasurableSpace γ} {f : α × β → γ} (hf : ∀ y, Measurable fun x => f (x, y)) :
     Measurable f :=
   measurable_from_prod_countable' hf (by simp (config := {contextual := true}))
 
+/-- For the version where the second space in the product is countable,
+see `measurable_from_prod_countable`.-/
+lemma measurable_from_prod_countable'' [Countable β] [MeasurableSingletonClass β]
```

Comment:
instead of primes, could you use `_left` and `_right`?

Reply:
This lemma is actually not used here anymore, so I have moved it to branch `measurable_from_prod_countable_right` in case @LorenzoLuccioli or @pitmonticone want to save it